doc/release/1.13.0-notes.rst

```diff
@@ -184,6 +184,18 @@ of arrays.
 
 It is similar to Matlab's square bracket notation for creating block matrices.
 
+Numpy may be built with relaxed stride checking debugging
+---------------------------------------------------------
+Setting NPY_RELAXED_STRIDES_DEBUG=1 in the enviroment when relaxed stride
+checking is enabled will cause numpy to be compiled with affected strides set
+to bogus values in order to help smoke out incorrect usage of strides in
+downstream projects.
```

Comment:
Might be worth explaining what a typical "smoke out" looks like, and specifying that "bogus" means "extremely large".

Could do with a comma after "bogus values", too.